.azure/scripts/process_metadata.py

```diff
@@ -41,6 +44,33 @@ def collect_metadata_and_thumbnails():
                     print(f"Warning: '{build_pack}' is not a valid buildPreset for directory: {meta_file}. Excluding from index.json.")
                     continue
 
+                # check if a quick deployable sample has a valid image URL
+                if 'Quick Deployable' in data.get('tags', []):
```

Comment:
Shall we take `Quick Deployable` to a constant as well? Let's reuse it in other places too.

Reply:
Should we use `imageUrl` instead of this tag to determine quick-deployable samples? If `imageUrl` presents, we get that as a deployable one, if not otherwise. there is no point in adding the URL if it's not quick-deployable, isn't it?

Reply:
+1. This will improve the performance as well. We can check the `imageUrl` first and then ensure the Quick deployable tag is available.